microsoft/testsuites/mshv/mshv_root_tests.py

```diff
@@ -79,6 +82,12 @@ def verify_mshvlog_is_active(
 
         assert_that(mshvlog_running).is_true()
 
+        # Check the size of mshvlog logfile
+        mshvlog_logfile_size = os.path.getsize(self.mshvlog_logfile)
+        mshvlog_logfile_size == 0:
+            log.error("mshvlog logfile is empty.")
```

Comment:
This line is not necessary, because if it's 0, the test case will be failed by below assertion.